### PR TITLE
Refine documentation

### PR DESCRIPTION
**Pull request by palmskog**
Document how to cleanly load the plugin and explain its commands in a more structured way using GitHub markdown facilities. Use more standard placement of files and generate some boilerplate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,37 +71,57 @@ make install
...
-Use the tactic t to solve proof obligations generated by the `Parametricity` command.
+When translating terms containing section variables or axioms,
+it may be useful to declare a term to be the translation of a constant:
+```coq
+Realizer <constant_or_variable> [as <name>] [arity <n>] := <term>.
+```
 
+Note that translating a term or module may lead to proof obligations (for some
+fixpoints and opaque terms if you did not import `ProofIrrelevence`). You can
+declare a tactic to solve such proof obligations:
```
Comment (proux01) [inline README.md:124]:
```suggestion
fixpoints and opaque terms if you did not import `ProofIrrelevence`). You need to
declare a tactic to solve such proof obligations:
```
The tactic is currently the only way to solve proof obligations (I once tried to fix the obligation mechanism but this requires some work (which would maybe be better invested in a reimplementation based on coq-elpi or metacoq)).

**Review by proux01 — APPROVED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (palmskog) [reply README.md]: OK, sounds good to me. I wasn't sure if there was a default tactic or not, like `Program` has a default tactic.

Comment (proux01) [reply README.md]: The main difference with `Program` is that the plugin doesn't properly register obligations, so when there is something left to prove, there is no other choice than resorting on the tactic (no interactive proof possibility).

**Pull request merged**